.github/scripts/compile.sh

```diff
@@ -130,6 +130,13 @@ do_build () {
   local suffix=${1}
   local runtime_check=${2}
 
+  if [ "$runtime_check" == "OFF" ]; then
+     # Turn on PGO only when we are not doing asan or tsan
+     BUILD_OPTS="-DENABLE_PROFILE_GUIDED_OPTIMIZATION=ON"
```

Comment:
Maybe not for this PR but for a subsequent one, but it seems useful to be able to turn off PGO. Also, possibly option to turn off the sanitizers as well would come handy.

Imagine wanting to try something with the image, and waiting for PGO, LTO, and sanitizers to build makes this slow to try things and iterate.

Reply:
> Maybe not for this PR but for a subsequent one, but it seems useful to be able to turn off PGO. Also, possibly option to turn off the sanitizers as well would come handy.

Agreed !

Reply:
@jiridanek I am going to merge this to main if you don't have any further comments. Please let me know. Thx

Reply:
Ship it! Or, scupper it!